modules/swagger-codegen/src/main/resources/htmlDocs2/sample_php.mustache

```diff
@@ -1,18 +1,18 @@
-
+<&#63;php
 require_once(__DIR__ . '/vendor/autoload.php');
 {{#hasAuthMethods}}{{#authMethods}}{{#isBasic}}
 // Configure HTTP basic authorization: {{{name}}}
-{{{invokerPackage}}}\Configuration::getDefaultConfiguration()->setUsername('YOUR_USERNAME');
-{{{invokerPackage}}}\Configuration::getDefaultConfiguration()->setPassword('YOUR_PASSWORD');{{/isBasic}}{{#isApiKey}}
+Swagger\Client\Configuration::getDefaultConfiguration()->setUsername('YOUR_USERNAME');
```

Comment:
@wing328 Sorry I didn't get back to you sooner. The problem is that the `{{{invokerPackage}}}` value is currently only correct for the Java example, but would not be accurate for any of the other languages. If you look at an existing example for PHP, you'll notice it's currently wrong (showing the default Java `invokerPackage` value):

![screen shot 2016-12-05 at 2 28 44 pm](https://cloud.githubusercontent.com/assets/4249514/20905067/305f2ec0-baf7-11e6-84d0-a1a159bd8f0a.png)

But, if I generate the `php` client, the README that's generated shows the following example:

![screen shot 2016-12-05 at 2 35 31 pm](https://cloud.githubusercontent.com/assets/4249514/20905350/45a7e8d4-baf8-11e6-8e3e-755303f18839.png)


Even if someone were to specify a custom `invokerPackage` command line build value for the htmlDocs2 template, it would only be valid for one of the six examples. 

Short of creating 6 different invoker package command line arguments, I don't see a good way to make them 100% accurate for each language.